### PR TITLE
Issue #SB-25751 fix:- hint,tip,score integration

### PR DESCRIPTION
**Pull request by rejneesh1**

### CODE REVIEW

File: projects/collection-editor-library/src/lib/services/editor/editor.service.ts [hunk 1]
```diff
@@ -283,7 +283,10 @@ export class EditorService {
     const instance = this;
     this.data = {};
     const data = this.treeService.getFirstChild();
-    const clonedNodeModified = _.cloneDeep(this.treeService.treeCache.nodesModified);
+    let clonedNodeModified = _.cloneDeep(this.treeService.treeCache.nodesModified)
+    for (const iterator in  clonedNodeModified) {
+      //  clonedNodeModified[iterator].metadata = _.omit(clonedNodeModified[iterator].metadata,['allowScoring'])
+    }
```
Comment (venkateshwarans) [inline projects/collection-editor-library/src/lib/services/editor/editor.service.ts:289]: please remove if not needed

File: projects/collection-editor-library/src/lib/components/question-option-sub-menu/question-option-sub-menu.component.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
       <input
         (change)="onValueChange($event, index)"
         class="sb-textbox false w-100 my-10"
-        [placeholder]="menu.name"
+        [placeholder]="menu.value ? menu.value : menu.name"
```
Comment (venkateshwarans) [inline projects/collection-editor-library/src/lib/components/question-option-sub-menu/question-option-sub-menu.component.html:20]: can be `menu.value || menu.name`

**Review by venkateshwarans — COMMENTED** [review]
check comments

**Comment by rejneesh1** [comment]
fixed

**Review by venkateshwarans — APPROVED** [review]

**Pull request merged**